ios/scripts/asc_max_build.py

```
@@ -0,0 +1,143 @@
+        with urllib.request.urlopen(req, timeout=30) as resp:
+            return resp.status, json.loads(resp.read() or b"{}")
+    except urllib.error.HTTPError as e:
+        return e.code, json.loads(e.read() or b"{}")
+
+
+def _resolve_app_id(token, bundle_id):
+    status, body = _api(
+        token, f"/v1/apps?filter[bundleId]={bundle_id}&fields[apps]=bundleId&limit=1"
+    )
+    if status != 200:
+        raise RuntimeError(f"apps lookup HTTP {status}: {json.dumps(body)[:300]}")
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)